x-pack/plugins/security_solution/cypress/e2e/data_sources/create_runtime_field.cy.ts

```diff
@@ -25,7 +27,7 @@ import { GET_TIMELINE_HEADER } from '../../screens/timeline';
 const alertRunTimeField = 'field.name.alert.page';
 const timelineRuntimeField = 'field.name.timeline';
 
-describe('Create DataView runtime field', () => {
+describe('Create DataView runtime field', { tags: [tag.ESS, tag.SERVERLESS] }, () => {
```

Comment:
My 2 cents... I feel like being explicit and verbose here isn't a bad thing. It makes things also very readable, especially for people outside of Security Solution who might not know the way we've set up things. They see both environments and it instantly makes sense what's happening. If we go the route of not having anything for shared tests, we'd have to document that it means, as some people might think it means the test aren't run at all (equivalent to `skip`).

Yes more files are being impacted but it's for this one PR only, once it's merged there won't be any unnecessary changes...

Reply:
Ok, I just started reviewing this PR. @YulNaumenko we touched on this at the meeting, but to answer your question here, the main idea was to be able to gradually opt in existing Cypress tests into the serverless suite. As you can read in the [comment](https://github.com/elastic/kibana/pull/162058#pullrequestreview-1549603969):

> The latter would allow us to:
> - tag all the existing Cypress tests as @ess in one PR
> - start gradually tagging tests as @serverless and increasing test coverage for it

My thinking was that if we don't do it like that, we'd opt in all our existing tests by default, and then would need to exclude those that are broken in serverless. It just seems riskier. If we opt in many tests at once, we may find in a few weeks that some of them are flaky in serverless. Are you suggesting to go this way?

++ to arguments of @PhilippeOberti, these look like good points to me as well 👍